Microsoft.Toolkit.Uwp.SampleApp/SamplePages/Mouse/MouseCursorPage.bind

```diff
@@ -0,0 +1,20 @@
+<Grid xmlns:ui="using:Microsoft.Toolkit.Uwp.UI.Extensions">
```

Comment:
We should use the xmlns shortnames consistently: ```xmlns:extensions="using:Microsoft.Toolkit.Uwp.UI.Extensions"```.

Reply:
Fixed in latest version.